src/umpire/strategy/NumaPolicyStrategy.hpp

```diff
@@ -0,0 +1,62 @@
+//////////////////////////////////////////////////////////////////////////////
+// Copyright (c) 2018-2019, Lawrence Livermore National Security, LLC.
+// Produced at the Lawrence Livermore National Laboratory
+//
+// Created by David Beckingsale, david@llnl.gov
+// LLNL-CODE-747640
+//
+// All rights reserved.
+//
+// This file is part of Umpire.
+//
+// For details, see https://github.com/LLNL/Umpire
+// Please also see the LICENSE file for MIT license.
+//////////////////////////////////////////////////////////////////////////////
+#ifndef UMPIRE_NumaPolicyStrategy_HPP
+#define UMPIRE_NumaPolicyStrategy_HPP
+
+#include <vector>
+
+#include "umpire/strategy/AllocationStrategy.hpp"
+
+#include "umpire/Allocator.hpp"
+
+// Forward declaration of bitmask for NUMA
+struct bitmask;
+
+namespace umpire {
+
+namespace strategy {
+
+class NumaPolicyStrategy :
```

Comment:
I think it's better to drop the `Strategy` from the name, less verbose.

Reply:
👍